app/forms/delivery_partner_form.rb

```diff
@@ -3,20 +3,22 @@
 class DeliveryPartnerForm
   include ActiveModel::Model
 
-  attr_accessor :name, :lead_providers, :provider_relationships
+  attr_accessor :name, :lead_providers, :provider_relationships, :provider_relationship_hashes
   validate :lead_providers_and_cohorts_validation
 
+  def self.provider_relationship_value(lead_provider, cohort)
+    "{\"lead_provider_id\": \"#{lead_provider.id}\", \"cohort_id\": \"#{cohort.id}\"}"
+  end
+
```

Comment:
This is pretty hacky. If anyone else can come up with a way to pre-populate nested checkboxes I'd love to hear it